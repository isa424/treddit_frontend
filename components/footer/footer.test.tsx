import React from 'react';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';
import { shallow } from 'enzyme';
import Footer from './footer';

describe('Footer component', () => {
	it('should match the snapshot', () => {
		const tree = renderer.create(<Footer />).toJSON();

		expect(tree).toMatchSnapshot();
	});

	it('should render', () => {
		const component = shallow(<Footer />);
		const el = component.find('#brand').first();

		expect(el).toBeTruthy();
		expect(el.text()).toEqual('Brand Â© 2021');
	});

	afterEach(cleanup);
});
