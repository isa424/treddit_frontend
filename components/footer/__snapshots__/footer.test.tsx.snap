// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match the snapshot 1`] = `
<footer
  className="bg-light border-top"
>
  <div
    className="footer__StyledFooter-x4cohs-0 jDTbAr d-flex justify-content-around container mt-4"
  >
    <ul
      className="footer__StyledList-x4cohs-1 bXwozA"
    >
      <li>
        <h4>
          About
        </h4>
      </li>
      <li>
        <a
          href="#"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Blog
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Advertising
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Careers
        </a>
      </li>
    </ul>
    <ul
      className="footer__StyledList-x4cohs-1 bXwozA"
    >
      <li>
        <h4>
          Help
        </h4>
      </li>
      <li>
        <a
          href="#"
        >
          Help
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Contact Us
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Rules
        </a>
      </li>
    </ul>
    <ul
      className="footer__StyledList-x4cohs-1 bXwozA"
    >
      <li>
        <h4>
          Apps & Tools
        </h4>
      </li>
      <li>
        <a
          href="#"
        >
          iOS
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Android
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Developers
        </a>
      </li>
    </ul>
    <ul
      className="footer__StyledList-x4cohs-1 bXwozA"
    >
      <li>
        <h4>
          Privacy & Settings
        </h4>
      </li>
      <li>
        <a
          href="#"
        >
          Privacy
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Terms & Conditions
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Content Policy
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Cookies
        </a>
      </li>
    </ul>
    <ul
      className="footer__StyledList-x4cohs-1 bXwozA"
    >
      <li>
        <h4>
          Social Links
        </h4>
      </li>
      <li>
        <a
          href="#"
        >
          Facebook
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Twitter
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Instagram
        </a>
      </li>
    </ul>
  </div>
  <div
    className="d-flex align-items-center justify-content-center py-3"
    id="brand"
  >
    Brand Â© 2021
  </div>
</footer>
`;
